tests/integration/test_recovery_time_metric/test.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from helpers.cluster import ClickHouseCluster
+
+cluster = ClickHouseCluster(__file__)
+node = cluster.add_instance(
+    "node",
+    main_configs=["configs/config.xml"],
+    with_zookeeper=True,
```

Comment:
`with_zookeeper` is redundant since there's embedded Keeper enabled in the server config